ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -102,6 +102,18 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
     Ok(views.html.recentactivity(userEventDBs))
   }
 
+  def statusDistribution = AuthAction { implicit request =>
+    val distribution: Map[RecipeStatus, Long] = Map(
+      New -> db.countRecipesInGivenStatus(New),
```

Comment:
Yep. I knew you would say that :) , but I didn't mind it this time because this is typically a call myself will make once every hour, or something.